x/delayedack/ibc_middleware.go

```diff
@@ -23,14 +20,16 @@ var _ porttypes.Middleware = &IBCMiddleware{}
 // IBCMiddleware implements the ICS26 callbacks
 type IBCMiddleware struct {
 	porttypes.IBCModule
-	keeper keeper.Keeper
+	keeper.Keeper
```

Comment:
not sure why u prefer to embed the keeper it this way
the `DelayedAck` keeper provides "store access" service to the middleware
IMO was more explicit and better readable before

Reply:
the idea was to be able to remove the IBC4 wrapper below, starting line 288